### PR TITLE
Makes circles have a correct 360 degrees again

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Changes the degrees a circle has from 365 to 360, major balance change

## Why It's Good For The Game
Idk the math guy said it worked or smt

## Images of changes
image

## Testing
I counted all degrees

## Changelog
tweak: Major balance change: 5 degrees have been removed from all circles

### CODE REVIEW

File: code/__HELPERS/type2type.dm [hunk 1]
```diff
@@ -126,7 +126,7 @@
 
 //Converts an angle (degrees) into an ss13 direction
 /proc/angle2dir(degree)
-	degree = ((degree+22.5)%365)
+	degree = ((degree + 22.5) % 360)
```
Comment (lewcc) [inline code/__HELPERS/type2type.dm:129]:
```suggestion
	degree = (degree + 22.5) % 360
```

**Review by lewcc — CHANGES_REQUESTED** [review]
Address these changes immediately or your pr will be closed

**Comment by DGamerL** [comment]
> Address these changes immediately or your pr will be closed

Ok yeah I don't want that, don't hurt me mister maintainer!

**Review by lewcc — APPROVED** [review]
mr maintainers rotate this man 365 degrees

**Comment by lewcc** [comment]
Shoutouts to https://github.com/Baystation12/Baystation12/pull/15761 as well for finding this issue a solid 8 years ago

**Pull request merged**